pandas/core/series.py

```diff
@@ -162,21 +164,27 @@ def __init__(self, data=None, index=None, dtype=None, name=None,
             elif isinstance(data, dict):
                 if index is None:
                     if isinstance(data, OrderedDict):
-                        index = Index(data)
+                        original_index = data.keys()
                     else:
-                        index = Index(_try_sort(data))
+                        original_index = _try_sort(data)
+
+                    index = Index(original_index)
```

Comment:
I wanted to use `_ensure_index`, but it does not create a `MultiIndex` when the dict keys are tuples.